src/app/screens/Send/index.tsx

```diff
@@ -41,6 +41,14 @@ function Send() {
 
       if (lnurl) {
         const lnurlDetails = await lnurlLib.getDetails(lnurl);
+        if (!("tag" in lnurlDetails)) {
```

Comment:
considering that we check for the tag further down, do we really need this if?

the issue was also only about handling the error case of the spec. 

so I'd do `if(lnurlDetails.satus == 'error' && lnurlDetails.reason) { `